### ISSUE TITLE
Find formula for price

### ISSUE THREAD

**Issue by chinhodado** [Heimdall409/persona4-fusion-calculator#3]
If this is the same as in Persona 4, a persona's compendium price is a function of its total stats.

In Persona 4, the cost for summoning a Persona, before any discounts are applied, is 

> 2000 + n^2, 

where n is the total number of stat points the registered Persona has (n = Strength + Magic + Endurance + Agility + Luck).

In Persona 3, I believe the formula is `2000 + 3n^2`.

If you can make a spreadsheet of the persona in your compendium with their prices and stats we can probably figure out the formula for Persona 5.

**Comment by Ashyaa** [comment]
This looks simple enough to do, but I question the utility of this formula in this project. 
In the current implementation, I don't support persona leveling nor stats sheets for each persona, and I don't plan to do it because it seems like a lot of work for, well, a little info.

What I can do though, once we figure out the price formula, is to figure out the base price of a persona, the price when the persona is at its lowest possible level (i.e. its rank). From that, I can compute a price for fusions with is the sum of the abse price of all the ingredients.

All in all, it will be pretty inaccurate from what a player will experience in the game: most of the time, he will have personae that are leveled up (then more expênsive), and might have to pay the Strength Coop Rank 10 fee to fuse a persona with a rank above his player level (even more expensive). 

To sum up, I can gladly provide you data to figure out the formula, but I don't promise any implementation of this without thinking a bit more about it, or without other opinions.

**Comment by chinhodado** [comment]
I see, can you provide me the data to figure out the formula then? Thanks.

**Comment by Ashyaa** [comment]
Here are is a few data for you to work on. Do post any results here, I am still interested in figuring out this formula, even if I'm not sure to use it!

Lvl 1 Arsène costs ¥1,806, n = 10:
![image](https://cloud.githubusercontent.com/assets/7237493/19836139/05aa2778-9e98-11e6-858b-23abc6fe2a53.png)

Lvl 8 Obariyon costs ¥2,639, n = 31:
![image](https://cloud.githubusercontent.com/assets/7237493/19836168/99b13bf0-9e98-11e6-81f4-7e46d34fba80.png)

Lvl 16 High Pixie costs ¥4,569, n = 55:
![image](https://cloud.githubusercontent.com/assets/7237493/19836184/ea3d5f7c-9e98-11e6-9ed2-f757f86b2bee.png)

Here are the prices if you want proof:
![image](https://cloud.githubusercontent.com/assets/7237493/19836186/05be3276-9e99-11e6-8725-cfb6a56b0f06.png)

One thing to note, there may be a discount applied. I don't remember it, but for Arsène to cost less than ¥2,000, I think it's very likely. In Persona 4, 50% of the compendium completed would reward you with a 15% discount. I tried a few things but did not figured out anything. I'll try to find more informations about the discounts first.
